3.6/data-modeling-databases-working-with.md

```diff
@@ -141,10 +141,13 @@ database must explicitly be requested by using the
 The *options* attribute can be used to set defaults for collections that will
 be created in the new database:
 
-* *sharding*: The sharding method to use. Valid values are: "" or "single
+* *sharding*: The sharding method to use. Valid values are: "" or "single".
+  Setting this option to "single" will enable the one-shard feature in the enterprise edition.
 * *replicationFactor*: Default replication factor. Special values include "satellite",
   which will replicate the collection to every DB-server, and 1, which disables replication.
-* *minReplicationFactor*: Default minimum replication factor.
+* *minReplicationFactor*: Default minimum replication factor. Collections with
+  a lower replication factor than this value can not be created in the new
```

Comment:
This is actually _not_ what this attribute is doing. 
Can you check other occurrences of "minReplicationFactor" in the docs or changelog or new features to see what it does exactly?